### PR TITLE
Add Python bindings

### LINKED ISSUE #1: Python bindings/wrapper
In the same way as we have them for the other languages

### PR DESCRIPTION
**Pull request by zoedberg**
closes #1

this is draft because first #22 needs to be merged (since in the demo I've used a method added by that PR).
so, as soon as that gets merged, I'll be able to rebase this branch

### CODE REVIEW

**Review by dr-orlovsky — APPROVED** [review]

**Comment by dr-orlovsky** [comment]
LGTM & ready to be merged after rebase (#22 is merged already)

**Comment by zoedberg** [comment]
@dr-orlovsky rebased

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**
